communication/communications.py

```diff
@@ -146,8 +148,18 @@ def left(self, speed):
   def right(self, speed):
       self.write(TEAM, RIGHT, int(speed))
 
+  def kick(self, speed):
+      self.write(TEAM, KICK, speed)
+
   def grab(self, speed):
       self.write(TEAM, GRAB, speed)
+  
+  def store(self, file):
+      bytes_to_store = len(file)
```

Comment:
use a context to open the file from disk, then pass in the content to the write functtion